tests/models/test_simplemodel.py

```diff
@@ -0,0 +1,31 @@
+from datetime import timedelta
+
+from telcell.data.parsers import parse_measurements_csv
+from telcell.models.simplemodel import pair_measurements_based_on_time,\
+    filter_delay
+
+
+def test_simplemodel(testdata_path):
+    tracks = parse_measurements_csv(testdata_path)
```

Comment:
In de `main` zit nu een fixture die de al ingelezen `Track`s terug geeft (genaamd `test_data`), dus dan kun je die direct gebruiken zodat je niet in elke test nog `tracks = parse_measurements_csv(testdata_path)` hoeft te typen.

Reply:
aangepast